config-templates/pgsql_tables.sql

```diff
@@ -0,0 +1,40 @@
+--Statistics for IdPs
+CREATE TABLE statistics (
+    year bigint NOT NULL,
+    month bigint NOT NULL,
+    day bigint NOT NULL,
+    sourceidp character varying(255) NOT NULL,
+    service character varying(255) NOT NULL,
+    count bigint,
+    PRIMARY KEY (year, month, day, sourceIdp, service)
+);
+
+CREATE INDEX statistics_i1 ON statistics (sourceIdp);
+CREATE INDEX statistics_i2 ON statistics (service);
```

Comment:
Do we need to explicitly create an INDEX for columns which are part of the table's primary key?

Reply:
The creation of indexes are included in the MySQL script [1], and I just "translated" the commands to be compatible with PostgreSQL.
Also, in rciam-2.0 we use indexes as well.

[1] https://github.com/nikosev/proxystatistics-simplesamlphp-module/blob/feature-rciam-changes-for-v3.2/config-templates/mysql_tables.sql#L9-L10